src/vma/util/asm-x86.h

```diff
@@ -36,6 +36,16 @@
 	dst += 8;			\
 	src += 8
 
+#define COPY_64B_NT_C(dst, src)	\
```

Comment:
The KVM environment can be detected in /proc/cpuinfo and configure can check the info. What about Xen support? AFAIK OFED support Xen as well.  BTW, if host OS is used to build for VM  then configure option is needed.   
@OphirMunk